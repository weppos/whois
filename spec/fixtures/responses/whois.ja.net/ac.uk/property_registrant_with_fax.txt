
Domain:
	ic.ac.uk

Registered For:
	Imperial College London

Domain Owner:
	Imperial College of Science, Technology and Medicine

Registered By:
	Imperial College of Science, Technology and Medicine

Servers:
	ns0.ic.ac.uk	155.198.142.80
	ns0.ic.ac.uk	2001:630:12:600:1::80
	ns1.ic.ac.uk	155.198.142.81
	ns1.ic.ac.uk	2001:630:12:600:1::81
	ns2.ic.ac.uk	155.198.142.82
	ns2.ic.ac.uk	2001:630:12:600:1::82
	authdns1.csx.cam.ac.uk	

Registrant Contact:
	Imperial College Hostmaster

Registrant Address:
	ICT
	Imperial College
	Exhibition Road
	London
	SW7 2AZ
	United Kingdom
	+44 20 7594 6965 (Phone)
	+44 20 7594 6958 (FAX)
	hostmaster@ic.ac.uk

Renewal date:
	Friday 14th Apr 2017

Entry updated:
	Saturday 14th March 2015

Entry created:
	Friday 7th November 2003

