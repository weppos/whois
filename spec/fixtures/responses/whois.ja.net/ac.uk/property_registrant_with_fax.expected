
#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  %s[0].id            == nil
  %s[0].name          == "Imperial College Hostmaster"
  %s[0].organization  == "Imperial College of Science, Technology and Medicine"
  %s[0].address       == "ICT\nImperial College\nExhibition Road\nLondon\nSW7 2AZ\nUnited Kingdom"
  %s[0].city          == nil
  %s[0].zip           == nil
  %s[0].state         == nil
  %s[0].country       == nil
  %s[0].country_code  == nil
  %s[0].phone         == "+44 20 7594 6965"
  %s[0].fax           == "+44 20 7594 6958"
  %s[0].email         == "hostmaster@ic.ac.uk"

